src/tjoperators.jl

```diff
@@ -0,0 +1,640 @@
+#= Operators that act on t-J-type models
+i.e. the local hilbert space consists of 
+
+- usual basis states: 
+    |∅⟩, |↑⟩, |↓⟩
+- slave-fermion basis states (c_σ = h† b_σ; holon h is fermionic, spinon b_σ is bosonic): 
+    |h⟩ = h†|∅⟩, |↑'⟩ = (b↑)†|∅⟩, |↓'⟩ = (b↓)†|∅⟩
+=#
```

Comment:
Can you move this information to the docstring of the `tj_space` function?
It seems quite useful, so it would be nice to have easy access.